CONTRIBUTING.md

```diff
@@ -44,6 +45,11 @@ mockrobiota/
 
 Each mock community directory contains the following directories and file types:
 
+### ``README.md``
+Lists a description of the dataset to appear in that dataset's home directory. At a minimum, should replicate the ``human-readable-description`` field of [``dataset-metadata.tsv``](#dataset-metadatatsv), but ideally should be expanded to provide as much detail as possible. See [``example-1``](https://github.com/caporaso-lab/mockrobiota/tree/master/data/example-1/README.md) for an example.
```

Comment:
I wonder if we should drop the ``human-readable-description`` field? It's prone to becoming outdated if it's replicated in two places. That could be part of a separate pull request, but it doesn't seem like a bad idea.